src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtAreasOfLawControllerTest.java

```diff
@@ -57,4 +60,19 @@ void retrieveCourtLocalAuthoritiesShouldReturnNotFoundForUnknownCourtSlug() thro
             .andExpect(content().string("Not found: " + TEST_SLUG));
     }
 
+    @Test
+    void updateCourtAreasOfLawShouldReturnUpdatedCourtAreasOfLaw() throws Exception {
+        final String expectedJson = getResourceAsJson(TEST_COURT_AREAS_OF_LAW_PATH);
+        final List<AreaOfLaw> areasOfLaw = asList(OBJECT_MAPPER.readValue(expectedJson, AreaOfLaw[].class));
+
+        when(adminService.updateAreasOfLawForCourt(TEST_SLUG, areasOfLaw)).thenReturn(areasOfLaw);
+
+        mockMvc.perform(put(BASE_PATH + TEST_SLUG + CHILD_PATH)
+                    .content(expectedJson)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .accept(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk())
+            .andExpect(content().json(expectedJson));
+    }
+
 }
```

Comment:
Think you can add a further test here for updating court areas of law with unknown court slug?

Reply:
adde dthe new unit test, thanks!